rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -843,6 +851,11 @@ def _watcher(rosa_cmnd, my_path, cluster_name_seed, cluster_count, delay, my_uui
             else:
                 logging.info("Waiting %d seconds for next watcher run" % delay)
                 time.sleep(delay)
+        elif os.path.isfile(file_path):
```

Comment:
If we are forcing this event we shouldn't bother dropping into the while loop and instead just trigger in the "if os.path.exists(file_path)" section